app/views/layouts/application.html.erb

```diff
@@ -27,9 +27,13 @@
     },
   } %>
 <% end %>
-
+<%
+  page_title = yield :title
+  content_title = page_title(content_item) || "GOV.UK"
```

Comment:
I think we can also combine L32-L33 as
`page_title = page_title.presence || page_title(content_item) || "GOV.UK"`